### ISSUE TITLE
Discrepencies between running delta inside lazygit and in the terminal

### ISSUE THREAD

**Issue by multivac61** [catppuccin/lazygit#49]
Thank you for a wonderful project.

I've been using it to rice my system thoroughly, it is a wonderful module.

Using `lazygit` today I noticed a discrepancy in the green/red background between using delta as its pager or in standalone. Why could that be?

![CleanShot 2024-09-03 at 19 18 59@2x](https://github.com/user-attachments/assets/69082db8-15c0-49d4-bac1-bcf064c198d9)

![CleanShot 2024-09-03 at 19 18 50@2x](https://github.com/user-attachments/assets/7724afc8-83f8-4e1b-8f07-5f8f9b8a617e)

[Here's the nixos config for home-manager (nix-darwin)](https://github.com/multivac61/nixos-config/blob/main/modules/shared/home.nix#L34)

[Related issue in github.com/catppuccin/nix](https://github.com/catppuccin/nix/issues/328)

**Comment by nullishamy** [comment]
If you can provide vanilla LG does this too, then we can open a report there, otherwise we'll (probably have to source dive to) set the right key in order for it to pick up the colours.

I am curious though, why exactly would you use a different pager with LG?

**Comment by multivac61** [comment]
Yeah, the vanilla LG version works as expected (see the red/green background in stock Alacritty):

![CleanShot 2024-09-05 at 15 20 17@2x](https://github.com/user-attachments/assets/3550d18b-62c8-4da2-9a4d-6fcd16217bad)

[I like how delta looks.](https://github.com/dandavison/delta#get-started)

**Comment by OJarrisonn** [comment]
I've also noticed the same issue, any updates here?

**Comment by nullishamy** [comment]
I haven't been able to investigate unfortunately as I don't use delta. As always life gets in the way, so I haven't been able to contribute to an awful lot of stuff. I aim to get back into it in the near future :tm:

**Comment by youyoumu** [comment]
same issue, and i'm using https://github.com/catppuccin/delta.
as an alternative, i slightly modified https://github.com/catppuccin/delta/blob/main/catppuccin.gitconfig
![image](https://github.com/user-attachments/assets/dffd553b-caf9-42be-8843-7ee2a3be6d3f)


before:
lazygit inside neovim
![image](https://github.com/user-attachments/assets/f553450d-7a08-4af5-9de1-086b78849613)
lazygit standalone
![image](https://github.com/user-attachments/assets/2ce3a0a3-caea-4195-a0bf-72f24e71fc54)

after:
lazygit inside neovim
![image](https://github.com/user-attachments/assets/4594d65f-ffcd-4d5e-b4fc-8a9b0c3251a6)
lazygit standalone
![image](https://github.com/user-attachments/assets/52528d9c-19c8-4afe-8f16-fccb26e8e709)
